### PR TITLE
Changelog for Mixxx 2.3.5

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,58 @@
...
+* Windows: Show a loopback device that allows to mix in system sound
+  [#11427](https://github.com/mixxxdj/mixxx/issues/11427)
+  [#11451](https://github.com/mixxxdj/mixxx/issues/11451)
+* Auto DJ: Fix Auto DJ indicator state when controlling it via  shortcut (SHIFT+F12)
+  [#11494](https://github.com/mixxxdj/mixxx/issues/11494)
+  [#11495](https://github.com/mixxxdj/mixxx/pull/11495)
+* Fix sorting via column header in external library features
+  [#11491](https://github.com/mixxxdj/mixxx/issues/11491)
+  [#11499](https://github.com/mixxxdj/mixxx/pull/11499)
+* Fix wrong background color on recent Linux distributions like Fedora 37
```
Comment (ronso0) [inline CHANGELOG.md:74]:
```suggestion
* Fix wrong waveform background color on recent Linux distributions like Fedora 37
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,58 @@
...
+  [#11393](https://github.com/mixxxdj/mixxx/issues/11393)
+* Fix crash on startup caused by faulty ASIO driver like FlexASIO 1.4 or Music Maker
+  [#11426](https://github.com/mixxxdj/mixxx/issues/11426)
+  [#10081](https://github.com/mixxxdj/mixxx/issues/10081)
+* Windows: Show a loopback device that allows to mix in system sound
+  [#11427](https://github.com/mixxxdj/mixxx/issues/11427)
+  [#11451](https://github.com/mixxxdj/mixxx/issues/11451)
+* Auto DJ: Fix Auto DJ indicator state when controlling it via  shortcut (SHIFT+F12)
+  [#11494](https://github.com/mixxxdj/mixxx/issues/11494)
+  [#11495](https://github.com/mixxxdj/mixxx/pull/11495)
```
Comment (ronso0) [inline CHANGELOG.md:70]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,6 +35,58 @@
...
+  [#11439](https://github.com/mixxxdj/mixxx/pull/11439)
+* Fix blank Waveform overview after changing Skin with a track loaded
+  [#11453](https://github.com/mixxxdj/mixxx/pull/11453)
+* Linux: Log a warning when the audio thread is not scheduled with real-time policy
+  [#11472](https://github.com/mixxxdj/mixxx/pull/11472)
+  [#11465](https://github.com/mixxxdj/mixxx/issues/11465)
+* Auto DJ: Fixes stop due to tracks with changed length
+  [#11479](https://github.com/mixxxdj/mixxx/pull/11479)
+  [#11492](https://github.com/mixxxdj/mixxx/pull/11492)
+  [#11448](https://github.com/mixxxdj/mixxx/issues/11448)
```
Comment (ronso0) [inline CHANGELOG.md:50]:
```suggestion
* Auto DJ: Fixes stop due to tracks with changed length
  [#11479](https://github.com/mixxxdj/mixxx/pull/11479)
  [#11492](https://github.com/mixxxdj/mixxx/pull/11492)
  [#11448](https://github.com/mixxxdj/mixxx/issues/11448)
  * Auto DJ: Fix Auto DJ indicator state when controlling it via  shortcut (SHIFT+F12)
  [#11494](https://github.com/mixxxdj/mixxx/issues/11494)
  [#11495](https://github.com/mixxxdj/mixxx/pull/11495)
```

**Review by ronso0 — COMMENTED** [review]
Thanks, just a small adding and some grouping (AutoDJ), otherwise LGTM

**Comment by daschuer** [comment]
I have rebased your suggestions.

**Comment by ronso0** [comment]
@mixxxdj/developers A second pair of eyes?

**Review by Swiftb0y — APPROVED** [review]
LGTM

**Pull request merged**